default/stringtables/en.txt

```diff
@@ -1725,6 +1725,56 @@ Network port to use for server discovery.
 OPTIONS_DB_NETWORK_MESSAGE_PORT
 Network port to use for client-server messaging.
 
+OPTIONS_DB_UI_DESIGN_PART_FILTER_DESC
+'''DesignWnd Redundant Part Filtering
+
+Deems a first part redundant to a second only if the second is all of:
+
+(i) more powerful
```

Comment:
what about otherwise equal but less expensive, or otherwise equal but faster to build parts?